Assets/Scripts/GameController.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.UI;
+
+public class GameController : MonoBehaviour
+{
+
+    bool hasPower = false;
+    public Canvas popupCanva;
+    private TMP_Text popupText;
+    public PlayerInput playerInput;

```

Comment:
```suggestion
    public PlayerInput PlayerInput;

```
Since we are using a capital letter (camel case) for public variables